credentials/apps/edx_credentials_extensions/edly_credentials_app/api/v1/views/program_certificate_configuration.py

```diff
@@ -0,0 +1,44 @@
+"""
+Views for Edly Program Certificate Configurations API
+"""
+from rest_framework import status, viewsets
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from credentials.apps.credentials.models import ProgramCertificate
+from credentials.apps.edx_credentials_extensions.edly_credentials_app.api.serializers import \
+    ProgramCertificateConfigurationSerializer
+
+
+class ProgramCertificateConfigurationViewSet(viewsets.ModelViewSet):
+    """
+    ViewSet for Program Certificate Configurations
+    """
+    lookup_field = 'program_uuid'
+    lookup_value_regex = '[0-9a-f-]+'
+    permission_classes = (IsAuthenticated,)
+    serializer_class = ProgramCertificateConfigurationSerializer
+    queryset = ProgramCertificate.objects.all()
+
+    def update(self, request, *args, **kwargs):
+        """
+        Update existing Program Certificate Configuration.
+        """
+        data = request.data.copy()
```

Comment:
Why are we creating a copy of the `request data`?

Reply:
The intention was to create a mutable version of data considering context-specific data but the serializer automatically discards other fields.